plotting/load_sn1a_data.py

```diff
@@ -36,6 +36,9 @@ def read_obs_data(path="observational_data"):
 
     hcorr = log10(h) - log10(0.7)  # h^-2 for SFR, h^-3 for volume
 
+    import os
```

Comment:
Why this change? as far as I can see cwd is unused.

Reply:
yep, another leftover from testing